addons/gamification/data/goal_base.xml

```diff
@@ -1,7 +1,7 @@
 <odoo>
     <!-- Mail template is done in a NOUPDATE block
          so users can freely customize/delete them -->
-    <data noupdate="1">
+    <data noupdate="0">
```

Comment:
Why oh why would you want to change a no update to an update? This'll break the whole logic of e-mail templates and updating your whole Odoo would revert possible made  changes.